.github/workflows/ci.yml

```diff
@@ -49,10 +49,9 @@ jobs:
 
       - name: Install latest nightly Rust toolchain
         if: steps.mdbook-cache.outputs.cache-hit != 'true'
-        uses: actions-rs/toolchain@v1
-        with:
-            toolchain: nightly
-            override: true
+        run: |
+          rustup update nightly
+          rustup override set nightly
```

Comment:
would doing `rustup default nightly` instead of these 2 commands result in a stale toolchain

Reply:
also, looking at CI log, it appears the override is not needed